Project.toml

```diff
@@ -37,7 +37,7 @@ Parameters = "0.10,0.11,0.12"
 RDatasets = "0.5,0.6"
 Reexport = "0.2"
 Requires = "1.0.1"
-SpecialFunctions = "0.10"
+SpecialFunctions = "≥ 0.10.0"
```

Comment:
For automerging, I think that we have to have a bounded range (not just a lowerbound). Maybe just do 0.10?